IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate/WebElementExtensions.cs

```diff
@@ -48,15 +48,44 @@ public static void ScrollIntoView(this IWebElement element, IWebDriver driver, i
             js.ExecuteScript($"window.scrollTo(0,{position})");
         }
 
+        /// <summary>
+        /// Clears the current text, 
+        /// uses SendKeys to send the specified value to the element, 
+        /// </summary>
+        /// <param name="value">Value to send to the element</param>
+        public static void ClearAndSendKeys(this IWebElement element, string value)
+        {
+            element.Clear();
+            element.SendKeys(value);
+            if (element.GetAttribute("value") != value)
+            {
+                element.Clear();
+                element.SendKeys(value);
+            }
+        }
+
+        /// <summary>
+        /// Clears the current text, 
+        /// uses SendKeys to send the specified value to the element, 
+        /// then tabs out of the field
+        /// </summary>
+        /// <param name="value">Value to send to the element</param>
+        public static void ClearAndSendKeysAndTab(this IWebElement element, string value)
```

Comment:
This seems like it would be better as an optional boolean parameters on the end of the previous method. Seeing double conjunctions in the method name is a code smell.

Reply:
Understood. And it may be another case of trying to solve too much. I'll revisit this week and see if it makes sense to keep around since... I mean really, you could do the same thing with the ClearAndSendKeys("some text" + Keys.Tab); We created a helper method specifically because Maximo needs you to tab out of a field before doing some processing on the UI, which... while it's not unheard of, it's not super duper common. So, again, may be that I'm trying to solve too much here.